### PR TITLE
Give Profile back its hover effect

### PR DESCRIPTION
**Pull request by PerrineBoissieres**
It's back! 

Capture d’écran 2019-06-17 à 15 48 44

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [...]
 
+- **[FIX]** `Profile` retrieves its hovering effect (similar to other clickable items)
```
Comment (Tuxosaurus) [inline CHANGELOG.md]: - retrieves
+ reacquires

Comment (PerrineBoissieres) [reply CHANGELOG.md]: I was searching for the word, thanks <3

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 [...]
 
+- **[FIX]** `Profile` reacquires its hovering effect (similar to other clickable items)
```
Comment (wakooka) [inline CHANGELOG.md]: `gets back` intead of `reacquires`

Comment (Bidjii) [reply CHANGELOG.md]: `intead` > `instead` 😬

Comment (wakooka) [reply CHANGELOG.md]: meta typo

Comment (PerrineBoissieres) [reply CHANGELOG.md]: No one agrees on how to write in english x)

**Pull request merged**